src/TimeTrack.cpp

```diff
@@ -270,7 +267,7 @@ bool TimeTrack::HandleXMLTag(const wxChar *tag, const wxChar **attrs)
    return false;
 }
 
-void TimeTrack::HandleXMLEndTag(const wxChar * WXUNUSED(tag))
+void TimeTrack::HandleXMLEndTag(const std::string_view&  WXUNUSED(tag))
```

Comment:
I like to clean up these WXUNUSED things when I have excuse to disturb a function.  Just omit the argument name.  Or, use `[[maybe-unused]]` if you want to be very modern.

Reply:
I don't like it either, but I've tried to leave as much of the original code as possible